docs/standard/microservices-architecture/architect-microservice-container-applications/using-azure-service-fabric.md

```diff
@@ -23,67 +23,68 @@ Service Fabric provides two broad areas to help you build applications that use
 
 ## Service Fabric and containers
 
-With regard to containers in Service Fabric, you can also deploy services in container images within a Service Fabric cluster. As Figure 4-27 shows, most of the time there will only be one container per service.
+With regard to containers in Service Fabric, you can also deploy services in container images within a Service Fabric cluster. As Figure 4-28 shows, most of the time there will only be one container per service.
 
-![](./media/image31.png)
+![](./media/image32.png)
 
-**Figure 4-27**. Business microservice with several services (containers) in Service Fabric
+**Figure 4-28**. Business microservice with several services (containers) in Service Fabric
 
 However, so-called “sidecar” containers (two containers that must be deployed together as part of a logical service) are also possible in Service Fabric. The important thing is that a business microservice is the logical boundary around several cohesive elements. In many cases, it might be a single service with a single data model, but in some other cases you might have physical several services as well.
 
-As of this writing (April 2017), in Service Fabric you cannot deploy SF Reliable Stateful Services on containers—you can only deploy guest containers, stateless services, or actor services in containers. But note that you can mix services in processes and services in containers in the same Service Fabric application, as shown in Figure 4-28.
+As of mid-2017, in Service Fabric you cannot deploy SF Reliable Stateful Services on containers—you can only deploy stateless services and actor services in containers. But note that you can mix services in processes and services in containers in the same Service Fabric application, as shown in Figure 4-29.
 
-![](./media/image32.png)
+![](./media/image33.png)
 
-**Figure 4-28**. Business microservice mapped to a Service Fabric application with containers and stateful services
+**Figure 4-29**. Business microservice mapped to a Service Fabric application with containers and stateful services
 
-Support is also different depending on whether you are using Docker containers on Linux or Windows Containers. Support for containers in Service Fabric will be expanding in upcoming releases. For up-to-date news about container support in Azure Service Fabric, see [Service Fabric and containers](https://docs.microsoft.com/azure/service-fabric/service-fabric-containers-overview) on the Azure website.
+For up-to-date news about container support in Azure Service Fabric, see [Service Fabric and containers](https://docs.microsoft.com/azure/service-fabric/service-fabric-containers-overview).
```

Comment:
nit: our boilerplate sentence for links like these would be "For more information about X, see [link]."

We can't really promise the article has up-to-date news.

Reply:
OK, I'll change it, but I do know that that article will have newer info about that support... ;)